### PR TITLE
Add Rakesh Gariganti as notation maintainer

### PR DESCRIPTION
**Pull request by vaninrao10**
Add Rakesh Gariganti as a seed maintainer of Notation based on [their](https://github.com/notaryproject/notation/commits?author=rgnote) activity as per - https://github.com/notaryproject/notation/issues/514

Signed-off-by: vaninrao10 <111005862+vaninrao10@users.noreply.github.com>

### CODE REVIEW

**Review by toddysm — APPROVED** [review]
LGTM but IANAM

**Review by priteshbandi — APPROVED** [review]
LGTM but IANAM

**Review by yizha1 — APPROVED** [review]
LGTM

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Comment by SteveLasker** [comment]
+1 to adding @rgnote 
Please see comments in https://github.com/notaryproject/notation/pull/530, https://github.com/notaryproject/notation/pull/531, https://github.com/notaryproject/notation/pull/532

**Review by JeyJeyGao — COMMENTED** [review]
LGTM but IANAM

**Review by patrickzheng200 — APPROVED** [review]
LGTM but IANAM

**Review by binbin-li — APPROVED** [review]
lgtm

File: MAINTAINERS [hunk 1]
```diff
@@ -0,0 +1 @@
+Rakesh Gokarn <garigant@amazon.com> (@rgnote)
```
Comment (gokarnm) [inline MAINTAINERS:1]:
```suggestion
Rakesh Gariganti <garigant@amazon.com> (@rgnote)
```

**Review by iamsamirzon — APPROVED** [review]

**Comment by yizha1** [comment]
Close as aggregated PR https://github.com/notaryproject/notation/pull/542 was merged

**Pull request closed without merging**